dev-libs/snowball-stemmer/snowball-stemmer-2.2.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MY_TESTDATA_COMMIT="0703f1d6a21802c3ff00c2c8b31bd255b74b2aec"
+
+DESCRIPTION="Snowball compiler and stemming algorithms"
+HOMEPAGE="https://snowballstem.org/ https://github.com/snowballstem/snowball/"
+SRC_URI="https://github.com/snowballstem/snowball/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	test? ( https://github.com/snowballstem/snowball-data/archive/${MY_TESTDATA_COMMIT}.tar.gz -> snowball-data-${MY_TESTDATA_COMMIT}.tar.gz )"
+
+LICENSE="BSD"
+SLOT="0/$(ver_cut 1)"
```

Comment:
I'm sorry to bother you one more time, but I wonder if you could run 'abidiff' (possibly using the iwdevtools bashrc wrapper) to see if it really has a stable ABI between e.g. 2.1.0 -> 2.2.0. I'm worried it doesn't, especially given upstream expect a static library, but it might be fine. They at least knew to increment the SONAME before, so..

Reply:
Sure, I'll check it.

Reply:
I am not sure if I did that correctly, but the ABI seems to be same
```sh
# qa-cmp -W /var/tmp/portage/dev-libs/snowball-stemmer-2.{1.0-r1,2.0}
CMP: dev-libs/snowball-stemmer-2.1.0-r1/image with dev-libs/snowball-stemmer-2.2.0/image
------> FILES(+0,-0) SONAME(+0,-0) ABI(nodebug) SIZE(+0.17%)
```
and
```sh
abidiff --verbose /var/tmp/portage/dev-libs/snowball-stemmer-2.1.0-r1/image/usr/lib64/libstemmer.so.2.1.0 /var/tmp/portage/dev-libs/snowball-stemmer-2.2.0/image/usr/lib64/libstemmer.so.2.2.0
```
does not return anything.